brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Default block design by designers.

### Description
- Task - https://app.startinfinity.com/b/Mi5zu3VRVzk/iHKt2gD8SSQ/dd3ef2f2-2811-42bc-b783-be3753c2f378?view=0a6f113f-f5c3-4b4e-82f0-1a6fc4831d3e&t=attributes
- Sheet - https://docs.google.com/spreadsheets/d/168ala6iLnIRmyKUXdXmZv5wEgDks0v8l25Q2baNcNMM/edit#gid=419838531

### Types of changes
Bugfix (non-breaking change which fixes an issue) 

### How has this been tested?

- Drag and drop block is mentioned in the above sheet.
- Check reported improvement by designers if it is fixed or not.

### Checklist:
